event-svc/src/event/service.rs

```diff
@@ -41,6 +41,7 @@ pub struct EventService {
     pub(crate) pool: SqlitePool,
     _validate_events: bool,
     delivery_task: DeliverableTask,
+    _ethereum_rpc_urls: Vec<String>,
```

Comment:
do we want the urls on the EventService or a ETH client struct that has all the ETH rpc logic in its functions and the urls in its state?

Reply:
From what I've seen of David's branches he has made a ETH client so up to him I think if this client should be provided to the service or created from the URLs in the service itself?